pkg/loadbalancer/experimental/bpf_reconciler.go

```diff
@@ -866,6 +900,31 @@ func (ops *BPFOps) releaseBackend(id loadbalancer.BackendID, addr loadbalancer.L
 	ops.backendIDAlloc.deleteLocalID(loadbalancer.ID(id))
 }
 
+func (ops *BPFOps) computeMaglevTable(svc *Service, bes []*BackendWithRevision) ([]loadbalancer.BackendID, error) {
+	backendsMap := make(map[string]*loadbalancer.Backend, len(bes))
```

Comment:
I would eventually want to get rid of this useless alloc&copy, but I think for now it's fine to keep `maglev.GetLookupTable` as is it is and then later revisit to clean this up.

Reply:
Yeah, makes sense.